### PR TITLE
Ticket 13030

### PR DESCRIPTION
**Pull request by aleksandra-tarkowska**
This should prevent python clients using OMERO.py gateway against older server where the following properties are not present.

To test:
 - go to trout/merge choose seabass and log in as non admin user
 - check if you were successfully logged in and there were no SecurityViolation exception thrown.
 - log out and log in to merge
 - check if interpolation is on in webviewer

cc: @sbesson @joshmoore @will-moore

### CODE REVIEW

File: components/tools/OmeroPy/src/omero/gateway/__init__.py [hunk 1]
```diff
@@ -1542,14 +1546,25 @@ def getInterpolateSetting(self):
...
         """
-        return (self.getConfigService().getConfigValue(
-                "omero.client.viewer.interpolate_pixels") or 'true')
+        try:
+            interpolate = (
+                toBoolean(self.getConfigService().getConfigValue(
+                    "omero.client.viewer.interpolate_pixels")) or True
+            )
+        except:
+            interpolate = False
```
Comment (will-moore) [inline components/tools/OmeroPy/src/omero/gateway/__init__.py]: The default value should be True, since that is the existing default behaviour.

**Comment by will-moore** [comment]
This works for allowing 5.1.4 webclient to log into 5.1.3 server (seabass).
One tiny suggestion: ```getInitialZoomLevel()``` should probably return the default value ```0``` when this fails.

**Comment by aleksandra-tarkowska** [comment]
Thanks @will-moore for testing. All the default values were now updated.

**Comment by will-moore** [comment]
OK, looks great. Good to merge (unless we want to wait for this to build on trout and test there? I've only tested locally).

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [OMERO-5.1-merge-push#1177](https://ci.openmicroscopy.org/job/OMERO-5.1-merge-push/1177/). See the [console output](https://ci.openmicroscopy.org/job/OMERO-5.1-merge-push/1177/consoleText) for more details.
Possible conflicts:
  - PR #4172 will-moore 'Size limit for download as jpg'
    - components/tools/OmeroWeb/omeroweb/decorators.py
  - PR #4189 aleksandra-tarkowska 'ticket 13018'
    - components/tools/OmeroWeb/omeroweb/decorators.py

**Comment by aleksandra-tarkowska** [comment]
https://ci.openmicroscopy.org/job/OMERO-5.1-merge-push/1178/ do not conflict

**Comment by will-moore** [comment]
Works fine on trout -> seabass. Good to merge.

**Comment by aleksandra-tarkowska** [comment]
thanks @will-moore

**Comment by aleksandra-tarkowska** [comment]
--rebased-to #4200

**Pull request merged**